src/main/java/com/artgallery/infrastructure/ArtRepositoryJson.java

```diff
@@ -47,10 +47,8 @@ public void addPainting(Painting painting) {
     @Override
     public void updatePainting(Painting painting) {
         List<Painting> paintings = getPaintings();
```

Comment:
to niepotrzebne

Reply:
nie rozumiem dlaczego to niepotrzebne, bo metoda nie przyjmuje listy paintingów

Reply:
jak usuwam to mi krzyczy, że nie ma wskazanej listy paintings